homeassistant/components/ohme/config_flow.py

```diff
@@ -0,0 +1,40 @@
+"""UI configuration flow."""
+
+from typing import Any
+
+from ohme import OhmeApiClient
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+
+from .const import CONF_EMAIL, CONF_PASSWORD, DOMAIN
+
+USER_SCHEMA = vol.Schema(
+    {vol.Required(CONF_EMAIL): str, vol.Required(CONF_PASSWORD): str}
+)
+
+
+class OhmeConfigFlow(ConfigFlow, domain=DOMAIN):
+    """Config flow."""
+
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> ConfigFlowResult:
+        """First config step."""
+
+        errors: dict[str, str] = {}
+
+        if user_input is not None:
+            await self.async_set_unique_id(user_input[CONF_EMAIL])
```

Comment:
email is not a good unique id, use `self._async_abort_entries_match({CONF_EMAIL: user_input[CONF_EMAIL]})`